SYCL/GroupAlgorithm/SYCL2020/exclusive_scan.cpp

```diff
@@ -3,6 +3,8 @@
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 
+// XFAIL: hip
```

Comment:
Should we mention the reason why they fail here? It is due to a selection of undefined symbols for `__spirv_*` functions, right?

Reply:
done